### DISCUSSION TITLE
Expand frigate storage with "mergerfs"

### DISCUSSION THREAD

**Discussion by ClearlyDazed** [blakeblackshear/frigate#12300] [Ask A Question]
### What is your question:

I am contemplating adding another HDD to expand the existing dedicated HDD storage for clips and recordings using "mergerfs". Frigate is running on a Linux system. Wondering if anyone has done this and if Frigate behaves normally wrt to storing and cleaning of recordings. Also would there be any ill effects with Home Assistant on a separate system running proxy Frigate app.

**Comment by NickM-27** [comment]
I use frigate with unraid and that uses fuse.fs, since frigate runs in docker it does not know or care what the underlying storage is using as long as it performs correctly
